### DISCUSSION TITLE
WYSIWYG, adding a link like www.something.com makes a broken link

### DISCUSSION THREAD

**Discussion by dsheiko** [directus/directus#5941] [Archive]
As an admin I have WYSIWYG  fields. When I add a link like www.something.com it's accepted literally and ends up as 
```
<a title="" href="www.something.com" target="_blank" rel="noopener">www.something.com</a>
``` 

Expected behavior: links starting with anything, but a single slash.. or at least any with `www` get prefixed with "//"
```
www.something.com => //www.something.com
```

Project details
Directus Version: 9.0.0-rc.69
Environment: production
OS: Ubuntu 18.04.2 LTS
Node: 11.15.1

**Comment by rijkvanzanten** [comment]
This is expected behavior. A link can be anything (doesn't have to be http(s) necessarily), so we can't just auto-prefix any non-http link with http://. Directus' TinyMCE configuration doesn't automatically convert pasted / entered links, so I'm assuming you were talking about the add-link dialog (cmd+K or toolbar), in which case you can set the href differently from the displayed value in the text:

<img width="533" alt="CleanShot 2021-05-28 at 12 05 36@2x" src="https://user-images.githubusercontent.com/9141017/120012126-19ead580-bfad-11eb-9004-d7f864766752.png">
